xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -284,15 +275,9 @@ void CGUIDialogPVRTimerSettings::InitializeSettings()
   if (tag->SupportsFolders())
     AddEdit(group, SETTING_TMR_DIR, 19076, 0, tag->m_strDirectory, true, false, 19104);
 
-  AddToggle(group, SETTING_TMR_RADIO, 19077, 0, tag->m_bIsRadio);
```

Comment:
I remember @ronie shouting at me some weeks ago never ever to remove a string because I cannot know whether it is used by some addon!?